packages/bygger/cypress/e2e/settings.spec.cy.ts

```diff
@@ -0,0 +1,45 @@
+import { NavFormType } from "@navikt/skjemadigitalisering-shared-domain";
+
+describe("FormSettingsPage", () => {
+  beforeEach(() => {
+    cy.intercept("GET", "/api/config", { fixture: "config.json" }).as("getConfig");
+
+    cy.intercept("GET", "/form?*", {
+      fixture: "getForm.json",
+    }).as("getForm");
+    cy.intercept("GET", "/mottaksadresse/submission", { fixture: "mottakadresse.json" }).as("getMottakAdresse");
+    cy.visit("forms/cypressinnstillinger/settings");
+  });
+
+  it("Fills all elements in settings page", () => {
+    cy.fixture("submitData.json").as("submitData");
+
+    cy.get("@submitData").then((mockSubmitData: any) => {
+      cy.intercept("PUT", "form/*", (req, res) => {
+        const submitData = mockSubmitData as NavFormType;
+        expect(req.body.properties.tema).to.include(submitData.properties.tema);
+        expect(req.body.title).to.include(submitData.title);
+        expect(req.body.properties.skjemanummer).to.include(submitData.properties.skjemanummer);
+        expect(req.body.properties.descriptionOfSignatures).to.include(submitData.properties.descriptionOfSignatures);
+        expect(req.body.properties.signatures[0].label).to.include(submitData.properties.signatures[0].label);
+        expect(req.body.properties.signatures[0].description).to.include(
+          submitData.properties.signatures[0].description
+        );
+        req.reply(submitData);
+      }).as("compareSubmitData");
+    });
+
+    //cy.get('a').contains("TEST-CYPRESS").click()
+    //cy.get('a').contains("Innstillinger").click()
+    cy.get('[id="title"]').focus().clear().type("Cypress test for settings page");
```

Comment:
Sjekk om det går an å bruke testing-library sin "get by name and role", der name = label?